### DISCUSSION TITLE
how to position profile at the beginning of mpv.conf?

### DISCUSSION THREAD

**Discussion by geextahslex** [mpv-player/mpv#14430] [Q&A]
Hi, I have a profile that I need to adjust often. Now it is at the end of the config file, like it is shown most of the time in other configs. So I have to scroll and find it every time. Is it possible to put it at the beginning before the default settings? How to "end" the profile, so he doesn't put everything that comes after it and interpret it as a profile?

I don't understand "_To end the profile, start another one or_ **use the profile name default to continue with normal options**"

Use the profile name `[default]` doesn't work

I hope this makes sense.

Thank you :)

**Comment by hooke007** [comment]
>I don't understand "To end the profile, start another one or use the profile name default to continue with normal options"

```
[your-profile1]
xxx=xxx
xxx=xxx
xxx=xxx

[default]
xxx=xxx
xxx=xxx
xxx=xxx
...

```

**Comment by hooke007** [comment]
BTW, I use https://mpv.io/manual/master/#options-include in a seperate file to contains all profiles.
